prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -1037,3 +1037,39 @@ periodics:
         requests:
           memory: 200Mi
           cpu: 80m
+
+- name: kyma-gke-compass-integration-periodic
+  cron: "00 00 * * *" # "At 00:00 every day"
+  skip_report: false
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20190528-8897828
+        securityContext:
+          privileged: true
+        command:
+          - "bash"
+        env:
+          - name: CLOUDSDK_COMPUTE_ZONE
+            value: "europe-west4-b"
+          - name: INPUT_CLUSTER_NAME
+            value: "compass-integration-periodic"
+        args:
```

Comment:
Please keep `args` close to `command` as they represent overall command that runs on that job.

Reply:
moved it